### PR TITLE
SR1.1.1 Add support for showing daily standard deviation on inflammation data

### PR DESCRIPTION
**Pull request by marinalexs**
Extend the graphs to show the daily standard deviation as required by SR1.1.1.

### CODE REVIEW

File: tests/test_models.py [hunk 1]
```diff
@@ -29,3 +31,13 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
```
Comment (laura-riedel) [inline tests/test_models.py:35]: wonderful parametrization! :D

**Review by laura-riedel — COMMENTED** [review]
great work

File: tests/test_models.py (hunk 1 above)
Comment (marinalexs) [reply tests/test_models.py:35]: Thank you!

File: inflammation/models.py [hunk 2]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
```
Comment (elena-rae) [inline inflammation/models.py:38]: variable name unclear, add details to function docstring?

File: inflammation/models.py [hunk 3]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
...
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
+    return {'standard deviation': s_dev2}
```
Comment (elena-rae) [inline inflammation/models.py]:
```suggestion
    return s_dev2
```
better return only standard deviation, not a dictionary

**Review by elena-rae — CHANGES_REQUESTED** [review]
suggested some changes in s_dev function

**Review by laura-riedel — APPROVED** [review]

**Pull request merged**